program/src/instruction.rs

```diff
@@ -13,14 +13,14 @@ use solana_program::{
 #[derive(Clone, BorshDeserialize, BorshSerialize, PartialEq, Debug)]
 pub struct CreateTokenAccount {
     /// Ethereum address
-    pub hashed_eth_pk: EthereumAddress,
```

Comment:
Also remove any `hashed_eth_pk` mentions from tests because now `cargo test-bpf` fails

Reply:
> Not sure that [this](https://github.com/atticwip/claimable-tokens/pull/3/files#diff-37970e7d817d4316a093bb967d74ff7e26d9c6c01d93f5a79d3d195ba9f6155bR284) validation is needed such as it's ethereum private key not Solana pubKey

What are you mean when say `this validation`?